### PR TITLE
Wsj_2mix transformer

### PR DESCRIPTION
**Pull request by simpleoier**
Previous PR was closed.

### CODE REVIEW

File: espnet/nets/pytorch_backend/e2e_asr_mix_transformer.py [hunk 1]
```diff
@@ -0,0 +1,452 @@
+#!/usr/bin/env python3
+# encoding: utf-8
+
+# Copyright 2019 Shigeki Karita
```
Comment (sw005320) [inline espnet/nets/pytorch_backend/e2e_asr_mix_transformer.py:4]: Could you summarize which part did you change from the original `espnet/nets/pytorch_backend/e2e_asr_transformer.py` with your authorship for this file?

Comment (simpleoier) [reply espnet/nets/pytorch_backend/e2e_asr_mix_transformer.py]: I will modify this. Thanks!

**Comment by simpleoier** [comment]
> * did you try specaugment? I'm just curious.

I did not use specaug. I just started one training process with `specaug` and may expect to get result 1 day later.

**Comment by sw005320** [comment]
> > * did you try specaugment? I'm just curious.
> 
> I did not use specaug. I just started one training process with `specaug` and may expect to get result 1 day later.

OK. You can do it in later PR.

File: espnet/nets/pytorch_backend/e2e_asr_mix_transformer.py (hunk 1 above)
Comment (sw005320) [reply espnet/nets/pytorch_backend/e2e_asr_mix_transformer.py]: Could you add more information of how it is different from `espnet/nets/pytorch_backend/e2e_asr_transformer.py`?

**Comment by sw005320** [comment]
Many thanks, @simpleoier!
